apps/mobile/src/screens/signin/signin.screen.tsx

```diff
@@ -1,29 +1,68 @@
-import { Button, View } from 'native-base';
+import { useNavigation } from '@react-navigation/native';
+import { Button, Heading, Input, Text, View } from 'native-base';
+import React from 'react';
+import { KeyboardAvoidingView } from 'react-native';
 
+import Routes from '../../routes';
 import AppStatusBar from '../../shared/components/status-bar/status-bar.component';
 import { useAuth } from '../../shared/hooks/use-auth-provider';
 import useLocale from '../../shared/hooks/use-locale';
+import theme from '../../theme';
 
 const SignInScreen = () => {
   const auth = useAuth();
   const { t } = useLocale();
 
+  const { navigate } = useNavigation();
+
   return (
-    <View flex="1" alignItems="center" justifyContent="center">
+    <View flex="1" padding={8}>
       <AppStatusBar />
-      <Button
-        variant="outline"
-        onPress={() => {
-          auth.signIn({
-            accessToken: '1234',
-            refreshToken: '4321',
-          });
-        }}
-      >
-        {t('SIGN_IN.FORM.LOGIN_BUTTON')}
-      </Button>
+      <KeyboardAvoidingView>
+        <View marginY={20}>
+          <Heading fontSize="4xl" color={theme.colors.primary[600]}>
+            Animavita
+          </Heading>
+          <Text fontSize="2xl">Sign In</Text>
```

Comment:
Please add this to the locale file. For now, we only have Portuguese translations